agentPresence/agentPresenceController.js

```diff
@@ -0,0 +1,87 @@
+var micc = new Micc('localhost');
+
+function login() {
+    micc = new Micc($('#miccServer').val());
+    micc.login(
+        $('#username').val(),
+        $('#password').val(),
+        connectToEmployeeHub
+    );
+}
+
+function connectToEmployeeHub(data) {
+    micc.getEmployeeStats('me', populateEmployeeInfo);
+    micc.getEmployeeBusyReasonCodes('me', populateBusyReasonCodes);
+
+    var miccSignalr = new MiccSignalR(data.miccServer, data.access_token);
+    miccSignalr.start(function () {
+        var hub = miccSignalr.createEmployeeHubProxy();
```

Comment:
Having more functions in miccsignalr.js is more valuable just like micc.js.  It makes it a good reference point for users with their own apps to look things up.  It separates out the sample code from the useful reference pieces that can be used elsewhere.  MiccSignalR.js should hide the hubs from the sample pieces.